utils/constants.lua

```diff
@@ -0,0 +1,26 @@
+local M = {}
+
+M.ABILITY_LIST = {"STR", "DEX", "CON", "INT", "WIS", "CHA"}
+
+M.SR_LIST = {"1/8", "1/4", "1/2", "1", "2", "3", "4", "5", "6", "7","8", "9", "10", "11", "12", "13", "14", "15"}
```

Comment:
Are all of these SRs actually valid SRs from Pokemon that exist? This **could** be parsed during the Python script that scrapes all the pokemon.

Reply:
Could be! But they are all used so no need I think